### DISCUSSION TITLE
Pair Extraordinaire achievement seems to not be updating correctly

### DISCUSSION THREAD

**Discussion by DenverCoder1** [community/community#37806] [Other Feature Feedback, Questions, & Ideas] [answered]
On my profile, it currently shows I have Level 3 of Pair Extraordinaire, it has shown this way since the first day achievements were released.

![image](https://user-images.githubusercontent.com/20955511/198892471-9c052104-df10-4db1-9020-93721da13ec3.png)

According to the links, it was first unlocked on May 30 and Level 3 unlocked on June 9. Based on [Schweinepriester/github-profile-achievements](https://github.com/Schweinepriester/github-profile-achievements), this means In a span of 10 days, I was co-authored in 24 merged pull requests. It is now 5 months later, and it has not updated despite that I have continued to co-author and get co-authored in pull requests at a similar rate, if not more than before.

Doing a search for ["Co-authored by Jonah Lawrence"](https://github.com/search?q=%22Co-authored+by+Jonah+Lawrence%22&type=commits) leads to over 150 results coming up, which are pretty much all in merged pull requests. This should be a **lower bound** for the number of commits I'm coauthored in since the achievement seems to include commits that I co-author others in as well.

According to the link above ([Schweinepriester/github-profile-achievements](https://github.com/Schweinepriester/github-profile-achievements)), Level 4 is unlocked with being coauthored in 48 merged pull requests.

If I have co-authored and been co-authored in hundreds of merged pull requests, why is it still showing Level 3, and not Level 4?

**Comment by jfullstackdev** [comment]
I suppose it's not anymore updating because of what happened in the past, particularly many users are abusing it, but I'm not very sure of that

**Comment by gonzalo-cordova-pou** [comment]
Something similar happening to me! Did you manage to solve it?

These are Achievements:
![image](https://user-images.githubusercontent.com/71346949/204637799-5e42d992-2c1a-414a-bc45-d41bd9fa583a.png)

These are from my colleague (He got them by co-authoring with me!!):
![image](https://user-images.githubusercontent.com/71346949/204637895-96dd6afc-b86a-4582-b36d-d0e5dc270727.png)

**Reply by jfullstackdev** [reply]
I saw the same post in the other thread, I think it was already answered there, the original author will receive the credits not the co-author

**Reply by DenverCoder1** [reply]
Based on the links in my achievement it seems to count both the author who co-authored and the one who was co-authored.

* https://github.com/nextcord/nextcord/pull/530
  * In this PR, I was co-authored by another user after leaving a review
* https://github.com/DenverCoder1/github-readme-streak-stats/pull/247
  * In this PR, I co-authored another user in a PR I opened

Both of these links appear as "unlock" milestones for me which seems to confirm that it should go both ways unless the requirements for the achievement were changed later on.

**Comment by DenverCoder1** [comment, marked as answer]
![image](https://github.com/community/community/assets/20955511/5344e979-6ec5-4a15-ab24-ef118d1dffb2)

I'm still not sure that it's counting correctly since the level 4 appears to have only been unlocked for me with a PR from 2 weeks and not sooner,  but it is showing level 4 for me now which is correct.

**Comment by jirafey** [comment]
i also encounter this problem 
i am way past 48 for 4x

**Comment by Paulitos** [comment]
My problem rn is that it's not showing in the frontend. Like the backend gets that I have a x2 but not the frontend. And I wonder why ngl.
